cachito/web/api_v1.py

```diff
@@ -72,6 +73,24 @@ def get_requests():
     # The call to `paginate` will inspect the current HTTP request for the
     # pagination parameters `page` and `per_page`.
     query = Request.query.order_by(Request.id.desc())
+    from_date = flask.request.args.get("from_date")
```

Comment:
@serg-cymbaluk is using a library to validate the very same inputs. Look at #574.
I believe there's the same code could be reused between both PRs.

Reply:
This is the relevante code: https://github.com/serg-cymbaluk/cachito/blob/a4bbcfbfa865f584d76c5ad37e355f2cb4427f30/cachito/web/api_v1.py#L723

It was already requested that the parameters' name were changed to match what you did here. 

Also, do mind that this lib will give out a validation error in case a `date` is passe to a `datetime` attribute, so there's the need of handling this to keep the current behavior.

Reply:
Done!